Test/git-issues/git-issue-2733.dfy

```diff
@@ -0,0 +1,12 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %dafny_0 /noVerify /compile:4 /compileTarget:cs "%s" >> "%t"
```

Comment:
Is it possible to use the new CLI UI here?

Reply:
Oh snap, I miss this comment.
Given that CI takes multiple hours I'd prefer to do it in a separate PR; OK with you?

Reply:
That'd be great. We can also do it later in one big fell swoop